Framework/Geometry/test/ComponentInfoBankHelpersTest.h

```diff
@@ -175,6 +175,46 @@ class ComponentInfoBankHelpersTest : public CxxTest::TestSuite {
     TS_ASSERT_THROWS(offsetFromAncestor(compInfo, ancestorIndex, currentIndex),
                      std::invalid_argument &);
   }
+
+  void test_isAncestorOf_finds_ancestor() {
+
+    auto instrument = ComponentCreationHelper::createMinimalInstrument(
+        V3D(0, 0, -10), V3D(0, 0, 0), V3D(0, 0, 10));
```

Comment:
```/* inline comments */``` for arguments for your test helpers please

Reply:
done